awx/playbooks/project_update.yml

```diff
@@ -190,6 +190,8 @@
   name: Install content with ansible-galaxy command if necessary
   vars:
     galaxy_task_env:  # configure in settings
+      # This environment variable is used for install roles
+      ANSIBLE_ROLES_PATH: "{{ projects_root }}/.__awx_cache/{{ local_path }}/stage/requirements_roles"
```

Comment:
This extra var is set here

https://github.com/ansible/awx/blob/86f6b54eec6758f618e5cb6d922f7ff508d75462/awx/main/tasks/jobs.py#L1281

and it doesn't merge, it just overwrites anything here. The only reason the variable was put here was to establish an expectation that the CLI ansible-playbook -e will define it.